config/webpack/development.js

```diff
@@ -24,9 +26,21 @@ module.exports = merge(sharedConfig, {
     publicPath: output.publicPath,
     compress: true,
     headers: { 'Access-Control-Allow-Origin': '*' },
-    historyApiFallback: true,
     watchOptions: {
       ignored: /node_modules/
+    },
+    historyApiFallback: true,
+    proxy: {
+      '/': {
+        target: `http://${settings.dev_server.host}:${env.PORT || '3000'}`,
+        secure: false
+      },
+      '/ws': {
+        target: `ws://${settings.dev_server.host}`,
```

Comment:
Missing port here?

Reply:
@himdel do we allow to customize the port for ws (vs wss)? is it set somewhere ?